packages/toolkit/package.json

```diff
@@ -4247,11 +4247,13 @@
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
         "buildScripts": "npm run generateClients && npm run generatePackage && npm run generateNonCodeFiles && npm run copyFiles",
         "browserWatch": "npm run clean && npm run buildScripts && webpack --config webpack.browser.config.js --watch",
+        "browserCompile": "npm run clean && npm run buildScripts && webpack --config webpack.browser.config.js",
```

Comment:
note: conflicts with https://github.com/aws/aws-toolkit-vscode/pull/4430

can we reference by --config-name instead of filepath, for future reference?

Reply:
Updated to do this